src/components/core/Card/Card.tsx

```diff
@@ -0,0 +1,38 @@
+import { cx } from "cva";
+import { ComponentInsightFragment } from "@/lib/contentful/fragments/ComponentInsightFragment";
+import { readFragment, FragmentOf } from "gql.tada";
+import { AssetFragment } from "@/lib/contentful/fragments/AssetFragment";
+import { Tag } from "@/components/core/Tag";
+import { Link } from "@/components/core/Link";
+
+type CardProps = {
+  data: FragmentOf<typeof ComponentInsightFragment>; // TODO: Add extra types when we define the articles, insights, etc.
+  className?: string;
+};
+
+export const Card = ({ data, className }: CardProps) => {
+  const { title, eyebrow, contentType, image, link } = readFragment(
+    ComponentInsightFragment,
+    data,
+  );
+  const imageUrl = readFragment(AssetFragment, image);
+
+  return (
+    <Link
+      link={link}
+      data-testid="Card"
+      className={cx(
+        "rounded-lg w-full h-full p-[1.3rem] overflow-hidden gradient-overlay bg-cover bg-center",
+        className,
+      )}
+      style={{ backgroundImage: `url(${imageUrl?.url})` }}
+    >
+      <p className="inline-flex mb-6 items-center gap-3">
+        <span className="block rounded-full h-2 w-2 bg-brand-300" />
+        <span className="typo-eyebrow">{eyebrow}</span>
+      </p>
+      <p className="mb-3">{title}</p>
+      {contentType && <Tag text={contentType} />}
```

Comment:
One small weird thing about this layout is that I believe these tags are supposed to be button/links themselves to a search of this content type filtered out by the tag you clicked. Problem being that you can't nest `<a>` tags inside of other `<a> tags.

To complicate this further, the images are supposed to zoom on hover.

I think you might have to make the link just wrap the text content and then have the background provided zoom in based on if you're hover/focus-visible of the [peer](https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-the-descendants-of-a-peer).